python/cugraph/dask/centrality/katz_centrality.py

```diff
@@ -0,0 +1,163 @@
+        it is internally calculated as 1/(degree_max) where degree_max is the
+        maximum out degree.
+        NOTE : The maximum acceptable value of alpha for convergence
+        alpha_max = 1/(lambda_max) where lambda_max is the largest eigenvalue
+        of the graph.
+        Since lambda_max is always lesser than or equal to degree_max for a
+        graph, alpha_max will always be greater than or equal to
+        (1/degree_max). Therefore, setting alpha to (1/degree_max) will
+        guarantee that it will never exceed alpha_max thus in turn fulfilling
+        the requirement for convergence.
+    beta : None
+        A weight scalar - currently Not Supported
+    max_iter : int
+        The maximum number of iterations before an answer is returned. This can
+        be used to limit the execution time and do an early exit before the
+        solver reaches the convergence tolerance.
+        If this value is lower or equal to 0 cuGraph will use the default
+        value, which is 100.
+    tolerance : float
+        Set the tolerance the approximation, this parameter should be a small
+        magnitude value.
+        The lower the tolerance the better the approximation. If this value is
+        0.0f, cuGraph will use the default value which is 1.0e-6.
+        Setting too small a tolerance can lead to non-convergence due to
+        numerical roundoff. Usually values between 1e-2 and 1e-6 are
+        acceptable.
+    nstart : dask_cudf.Dataframe
+        GPU Dataframe containing the initial guess for katz centrality
+        nstart['vertex'] : dask_cudf.Series
+            Contains the vertex identifiers
+        nstart['values'] : dask_cudf.Series
+            Contains the katz centrality values of vertices
+    normalized : bool
+        If True normalize the resulting katz centrality values
+
+    Returns
+    -------
+    PageRank : dask_cudf.DataFrame
+        GPU data frame containing two dask_cudf.Series of size V: the
+        vertex identifiers and the corresponding katz centrality values.
+
+        ddf['vertex'] : dask_cudf.Series
+            Contains the vertex identifiers
+        ddf['katz_centrality'] : dask_cudf.Series
+            Contains the katz centrality of vertices
+
+    Examples
+    --------
+    >>> import cugraph.dask as dcg
+    >>> Comms.initialize(p2p=True)
+    >>> chunksize = dcg.get_chunksize(input_data_path)
+    >>> ddf = dask_cudf.read_csv(input_data_path, chunksize=chunksize,
+                                 delimiter=' ',
+                                 names=['src', 'dst', 'value'],
+                                 dtype=['int32', 'int32', 'float32'])
+    >>> dg = cugraph.DiGraph()
+    >>> dg.from_dask_cudf_edgelist(ddf, source='src', destination='dst',
+                                   edge_attr='value')
+    >>> pr = dcg.katz_centrality(dg)
+    >>> Comms.destroy()
+    """
+
+    nstart = None
+
+    client = default_client()
+
+    input_graph.compute_renumber_edge_list()
```

Comment:
you state that the input graph could be from NetworkX, but calls will fail if it is a Nx DiGraph.

Also, Nx should not be used in MNMG

Reply:
updated doc